contact.html

```diff
@@ -0,0 +1,153 @@
+      <li><a href="blogs.html">Blog</a></li>
+
+    </ul>
+  </nav>
+
+
+  <!-- Signout page form using flex-->
+<form class="form-contact" method="get">
+  <div class="form-container" id="form">
+    <div class="container-content">
+      <h1>Checkout</h1>
+      <label for="name">Name</label>
+      <input type="text" id="name">
+    </div>
+    <div class="container-content">
+      <label for="email">Email</label>
+      <input type="text" id="email">
+    </div>
+    <div class="container-content">
+      <label for="street">Street Address</label>
+      <input type="text" id="street">
+    </div>
+
+
+    <div class="container-content">
+      <div class="">
+        <div class="city">
+          <label for="city">City</label>
+          <input type="text" id="city">
+        </div>
+        <div class="state-item">
+          <label for="state">State</label>
+          <select name="state" id="state-select">
+            <option value="">Please Select a State</option>
+            <option value="washington" >Washington</option>
+            <option value="montana">Montana</option>
+          </select>
+        </div>
+        <div class="zip">
+          <label for="zip">Zip Code</label>
+          <input type="text" id="zip">
+        </div>
+      </div>
+
+
+    </div>
+    <div class="container-content">
+    <div class="first-time-adopter">
+      First Time Adopter?
+    </div>
+      <div class="radio-adopt">
+        <input type="radio" id="radio1" name="firsttimeadopter" value="yes">
+        <label for="yes">Yes</label>
+        <input type="radio" id="radio2" name="firsttimeadopter" value="no">
+        <label for="no">No</label>
+      </div>
+      <!-- <div class="radio-adopt" id="input-2">
+        <input type="radio" id="radio2" name="input" value="no">
+        <label for="no">No</label>
+      </div> -->
+    </div>
+    <div class="container-content">
+      <label for="pickup">Pickup Location</label>
+    <select class="pickup" name="pick" id="pickup">
+      <option value="redmond">Redmond</option>
+      <option value="bothell">Bothell</option>
+      <option value="bellevue">Bellevue</option>
+    </select>
+    </div>
+  </div>
+
+
+  <div class="dogs-adopted-signout">
+    <h2>Dogs Adopted</h2>
+  </div>
+
+
+  <div class="lucduff">
+
+
+    <div class="lucas-img">
+      <img src="images/lucas-card.jpg" alt="black barking dog">
+    </div>
+    <div class="lucas-content">
+      <h5>Lucas</h5>
+      <p>Cost to Adopt: $123.45</p>
+    </div>
+    <div class="duffy-img">
+      <img src="images/duffy-card.jpg" alt="black barking dog">
+    </div>
+    <div class="duffy-content">
+      <h5>Duffy</h5>
+      <p>Cost to Adopt: $123.45</p>
+    </div>
+
+  </div>
+  <div class="total-adoption">
+    <h2>Total Adoption Cost</h2>
+    <p>$1234.56</p>
+  </div>
+  <div class="checkout-button">
+    <button type="submit" name="button" id="btn" onclick="getInfo()">submit</button>
```

Comment:
changing type="submit", to type="button" should keep the button from submitting the form and refreshing the page.